.github/workflows/ci.yaml

```diff
@@ -0,0 +1,36 @@
+on:
+    push:
+        branches-ignore:
+            - 'github-comments'
+    pull_request:
+        branches-ignore:
+            - 'github-comments'
+
+name: CI
+
+jobs:
+    build:
+        name: Build
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/checkout@v1
```

Comment:
`actions/checkout@v2` should be used instead -> https://github.com/actions/checkout/releases